fitbenchmarking/controllers/matlab_mixin.py

```diff
@@ -36,6 +38,7 @@ def __init__(self, cost_func):
                                        'fitting, module "dill" is required.')
 
         self.initial_params_mat = None
+        self.incompatible_minimizers.append('mantid')
```

Comment:
Should this be `incompatible_problems`? And also I noticed MatlabMixin doesn't inherit from the base controller, so maybe this would be better off as a class variable in the `MatlabController`, `MatlabStatController`, etc. classes. Unless there is a reason for it to go here?

Reply:
Good spot! I put a test in but forgot the matlab ones aren't running yet. I put it here as it covers all of the matlab based ones, so if we add more it's one less thing to get right. That said I'm happy to move it if you think it would be clearer.

The way the inheritance works, is that the MatlabController will inherit from the BaseController, and then from the MatlabMixin, so provided the mixin is used correctly it should be the same functionality.

Reply:
I see, so would this only work if the `BaseController` and `MatlabMixin` is inherited in the correct order? I'm thinking it might be safer to have `incompatible_problems` separately in each Matlab Controller class, but having said that I have no great preference so I'm happy with this if if you are

Reply:
Yes, that's true although I think the initialiser might not work if they're out of order, so we do need them to be in the correct order. Either way I'll move it to the individual controllers to make it more obvious